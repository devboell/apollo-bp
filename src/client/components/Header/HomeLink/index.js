import React from 'react'

import { Link } from 'react-router-dom'
import homeImage from './images/home.png'
import Image from './Image'

const HomeLink = () =>
  <Link to="/" >
    <Image src={homeImage} alt="Home" />
  </Link>

export default HomeLink
