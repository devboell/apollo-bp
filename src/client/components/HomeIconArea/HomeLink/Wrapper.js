import styled from 'styled-components'

const Wrapper = styled.div`
  margin-left: 20px;
  flex-basis: 15%;
`

export default Wrapper
