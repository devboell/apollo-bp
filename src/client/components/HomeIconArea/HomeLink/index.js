import React from 'react'
import { NavLink } from 'react-router-dom'

import homeImage from './images/home.png'
import Image from './Image'

const HomeLink = () =>
  <NavLink to="/" >
    <Image src={homeImage} alt="Home" />
  </NavLink>

export default HomeLink
