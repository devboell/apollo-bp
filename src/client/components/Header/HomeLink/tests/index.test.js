import React from 'react'
import HomeLink from '../index'

it('HomeLink component, snapshot', () => {
  const wrapper = shallow(<HomeLink />)
  expect(wrapper).toMatchSnapshot()
})
