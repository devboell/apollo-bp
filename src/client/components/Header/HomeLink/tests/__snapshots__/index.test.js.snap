// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeLink component, snapshot 1`] = `
<Link
  replace={false}
  to="/"
>
  <styled.img
    alt="Home"
    src="test-file-stub"
  />
</Link>
`;
